package.json

```diff
@@ -62,6 +62,7 @@
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
+    "less": "^2.7.1",
```

Comment:
What version of node and npm are you using? I think this is something that was resolved in later versions of npm.